examples/torchfx/builder.py

```diff
@@ -10,8 +10,9 @@
 to convert the output from the tracer into MLIR using the `torch`
 dialect.
 
-This file is licensed under a pytorch-style license
-See frontends/pytorch/LICENSE for license information.
+Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
nit: would recommend keeping the license in `#` style comments at the top, followed by the docstring.

(thanks for the docstrings btw :) )